### DISCUSSION TITLE
Collision bug?

### DISCUSSION THREAD

**Discussion by Sjokofant** [cyberbotics/webots#2573] [General]
Hi, 
Im getting what i think is a red collisions field when my robot gets to close to an object. Is it possible to make it non visible in webots?
![collision field](https://user-images.githubusercontent.com/28389296/102208479-c5a4db00-3ecf-11eb-8591-94e13d8ba332.PNG)

**Comment by omichel** [comment]
That looks strange. There is no such "red collision field" in Webots. Can you better explain what is this red object? Did you create it?

**Reply by Sjokofant** [reply]
It might be when the two bounding boxes touches, but I don’t know and it is not something I have created. Also, the field is not visible from the front.

**Reply by omichel** [reply]
It might be a laser beam... I suppose your robot has a [DistanceSensor](https://cyberbotics.com/doc/reference/distancesensor) node of type "laser". If you turn it to "generic" instead of "laser", you should be able to get rid of that red blob.

**Reply by Sjokofant** [reply]
Ha! that solved it. thanks!
